filebeat/module/elasticsearch/log/_meta/fields.yml

```diff
@@ -0,0 +1,301 @@
+- name: log
+  type: group
+  description: >
+    Contains fields from the Elasticsearch standard logs.
+  fields:
+    - name: component
+      description: >
+        The component within Elasticsearch that generated this event.
+    - name: reason
+      description: >
+        Extra reason or details for a particular event.
+    - name: exception
+      type: text
+      description: >
+        An exception trace for this event.
+    - name: allocation_info
+      type: text
+      description: >
+        Detailed cluster allocation information.
+    - name: cluster
+      type: group
+      description: >
+        Contains fields related to an Elasticsearch cluster.
+      fields:
+        - name: discovery_type
+          description: >
+            The discovery method this cluster is using.
+        - name: license_id
+          description: >
+            The license ID for this cluster.
+        - name: license_type
+          description: >
+            The type of license used on this cluster (i.e., trial, platinum, etc.)
+        - name: license_status
+          description: >
+            The license status (i.e., validity).
+        - name: action
+          description: >
+            A cluster-level action.
+        - name: action_details
+          type: text
+          description: >
+            A cluster-level action details.
+        - name: state
+          type: group
+          description: >
+            Contains fields related to the cluster state.
+          fields:
+            - name: version
+              type: long
+              description: >
+                The cluster state version.
+            - name: uuid
+              description: >
+                A cluster state UUID.
+            - name: diff
+              description: >
+                A diff size of the cluster state version.
+        - name: health
+          type: group
+          description: >
+            Contains fields related to the cluster health.
+          fields:
+            - name: old_value
+              description: >
+                The previous health of the cluster.
+            - name: new_value
+              description: >
+                The new health of the cluster.
+        - name: setting
+          type: group
+          description: >
+            Contains fields related to cluster settings.
+          fields:
+            - name: action
+              description: >
+                The action being applied to this cluster setting.
+            - name: name
+              description: >
+                The cluster setting name.
+            - name: old_value
+              description: >
+                The previous value of the setting.
+            - name: new_value
+              description: >
+                The new value of the setting.
+    - name: node
```

Comment:
In a second step we should unify this with the data from metricbeat: https://github.com/elastic/beats/blob/master/metricbeat/module/elasticsearch/node/_meta/fields.yml#L1 So if someone searches for `elasticsearch.node.name` for example he gets information for logs and metrics.

Reply:
This is a good idea! I will work on adopting the established names in Metricbeat into this module.